### PR TITLE
Adding initial implementation of materialize (writing off a day)

### PR DESCRIPTION
**Pull request by philippQubit**

### CODE REVIEW

File: mlviews/core.py [hunk 1]
```diff
@@ -81,17 +81,28 @@ def refresh_template_views(bq_client=None):
...
+
+
+def materialize_template_views(bq_client=None):
+    """
+    writes out the last day of data into a project bucket
+    """
+    if not bq_client:
+        bq_client = bq.build_client()
+
+    project_id = os.environ.get('PROJECT')
```
Comment (dkruchinin) [inline mlviews/core.py]: Should we do something if the env is not set?

Comment (mmniaziqb) [reply mlviews/core.py]: Maybe we should move `project_id = os.environ.get('PROJECT')` as its being done 4 times in this file

File: mlviews/core.py [hunk 2]
```diff
@@ -81,17 +81,28 @@ def refresh_template_views(bq_client=None):
...
         sqls.append(get_template_sql(t.name, tracking_id))
 
-    # replace all views
-    # includes no ops where compiled sql is equivalent to existing view sql
-    results = view_creator.create_or_replace_views(tables, sqls)
-    return results
+    return sqls
+
+
+def materialize_template_views(bq_client=None):
```
Comment (mmniaziqb) [inline mlviews/core.py]: `def materialize_template_views(bq_client=bq.build_client()):` will look more cool :)

File: mlviews/core.py (hunk 1 above)
Comment (philippQubit) [reply mlviews/core.py]: Default project if not set ... but will refactor to remove repetition of querying from os.env

**Pull request merged**